composer.json

```diff
@@ -4,21 +4,24 @@
   "minimum-stability": "stable",
   "prefer-stable": true,
   "require": {
-    "php": ">=8.0",
+    "php": ">=8.1",
     "ext-ctype": "*",
     "ext-iconv": "*",
     "doctrine/doctrine-bundle": "^2.8",
     "doctrine/doctrine-migrations-bundle": "^3.2",
     "doctrine/orm": "^2.14",
     "easycorp/easyadmin-bundle": "^4.5",
     "nelmio/security-bundle": "^3.0",
+    "surfnet/stepup-saml-bundle": "dev-feature/support-sf6-dependencies",
```

Comment:
normaal gesproken is het geen goed idee om deps die in `dev` status te gebruiken, maar aangezien jullie deze package zelf in beheer hebben lijkt me dat hier prima